### ISSUE TITLE
choose one not appering

### ISSUE THREAD

**Issue by manthanabc** [deepjyoti30/ytmdl#82]
i finally got it running but now when ever i enter any song it says not found

**Comment by deepjyoti30** [comment]
@manthanabc That happens sometimes, can you try running with a different song name. Sometimes ```ytmdl``` just out of the blue throws that error of ```not found``` but usually that happens every now and then.

Also can you share the steps, how you got it running.

**Comment by manthanabc** [comment]
No its happening literally everyone but I found that my YouTube dl is also giving the video not found error evrytime so I will need to reinstall it
> Also can you share the steps, how you got it running.
Oh ha a would love to do that I had installed ffmpeg installed with pip so I also manually installed that and added to path and then reinstalled this api and it works

**Comment by manthanabc** [comment]
> No its happening literally everyone but I found that my YouTube dl is also giving the video not found error evrytime so I will need to reinstall it
> 
> > Also can you share the steps, how you got it running.
> > 
Oh ha a would love to do that I had installed ffmpeg installed with pip so I also manually installed that and added to path and then reinstalled this api and it works

**Comment by deepjyoti30** [comment]
@manthanabc Can you updated ```youtube-dl``` to the latest version by doing ```pip install youtube-dl --upgrade --no-cache-dir```?

Try running it after that.

**Comment by JavaCafe01** [comment]
@deepjyoti30 I'm also getting this issue, even after updating `youtube-dl`.

**Comment by spikecodes** [comment]
Same exact issue for any song name:
![image](https://user-images.githubusercontent.com/19519553/86863752-a97b5300-c080-11ea-8c66-79b5530541a9.png)

**Comment by mementomori74** [comment]
im running in to the same issue

**Comment by deepjyoti30** [comment]
I just checked guys. The issue was occuring because ```ytmdl``` scraps YouTube's page in order to get the search results but recently YouTube made changes due to which most pages after scrapped, are returning nothing.

I redefined the search logic and made the move to use [youtube-scrape](https://github.com/HermanFassett/youtube-scrape)

Should be fixed in 72b51d4

You guys can test it and report back here if you find any issues.

I'll probably make a release soon since this patch is necessary.

**Comment by mementomori74** [comment]
tnx for your work!

**Comment by deepjyoti30** [comment]
@mementomori74 Really?? What is the issue? I thought it was working.

**Comment by spikecodes** [comment]
> I just checked guys. The issue was occuring because `ytmdl` scraps YouTube's page in order to get the search results but recently YouTube made changes due to which most pages after scrapped, are returning nothing.
> 
> I redefined the search logic and made the move to use [youtube-scrape](https://github.com/HermanFassett/youtube-scrape)
> 
> Should be fixed in [72b51d4](https://github.com/deepjyoti30/ytmdl/commit/72b51d472b8d79ce996144eee0fc4a8ebef34b46)
> 
> You guys can test it and report back here if you find any issues.
> 
> I'll probably make a release soon since this patch is necessary.

Thanks! `sudo python setup.py install` gives an error about Python 2 being unsupported so instead I ran 
```bash
git clone https://github.com/deepjyoti30/ytmdl && cd ytmdl && pip install .
```

and then tested it with
```bash
ytmdl "Roxanne by Arizona Zervas" --song ROXANNE --artist "Arizona Zervas" -q --nolocal
```
(`ytmdl "Roxanne by Arizona Zervas" -q --nolocal` had the wrong metadata originally) and it worked! 

![image](https://user-images.githubusercontent.com/19519553/86961139-96669280-c115-11ea-9d35-e4bafe4b0a94.png)

Thanks for your help and quick response! Looks ready for a new release!

**Comment by deepjyoti30** [comment]
Fixed in latest release.
